src/services/content.ts

```diff
@@ -1,10 +1,14 @@
 import i18n from 'services/i18n'
 import contentManifest from 'content/manifest.json'
 
-export const getManifestFilesFromClasses = classes =>
-  contentManifest.filter(file => classes.indexOf(file.class) !== -1)
+export const getManifestFilesFromClasses = (classes: string[]) =>
+  classes
+    .flatMap(className =>
+      contentManifest.filter(file => file.class === className)
```

Comment:
only thing I'd say is to switch the filter for a find to speed it up a little

Reply:
find only returns one item, here we want to return all items that match the class, so unfortunately `find` doesn't work for this